draft-frindell-webtrans-devious-baton.md

```diff
@@ -86,25 +86,32 @@ requires the server to interpret that as the default value.
 
 The server MUST support the following optional query parameters:
 
-* version - an integer specifying the draft version of Devious Baton the client
-  intends to use
+* `version` - an integer specifying the draft version of Devious Baton the
+  client intends to use
 
 If the version is invalid or the server does not support the specified version,
 it MUST reject the WebTransport session with a 4xx status code.  The default
 value is 0.
 
-* baton - an integer between 0 and 255, inclusive, which the server will use as
-  the initial baton value
+* `baton` - an integer between 1 and 255, inclusive, which the server will use
+  as the initial baton value
 
 If the baton value is invalid, the server MUST reject the WebTransport session
 with a 4xx status code.  There is no default - if unspecified the server chooses
-a random baton value.
+a random baton value between 1 and 255, inclusive.
 
-* count - an integer specifying how many batons will be sent in parallel
+If `count` is greater than 1, the baton parameter MAY be specified more than
+once to set the initial value for each baton.  If the number of baton parameters
+is greater than `count`, the server SHOULD ignore the extra parameters.  If the
+number of baton parameters is less than `count`, the server chooses a random
+baton value between 1 and 255, inclusive, for each baton without a value.
+
```

Comment:
Sorry, but that feels like gratuitous complexity. Having just one baton and running a copy of the same baton in all lanes is much simpler. In particular, consider the "receiving first baton" scenario. If there is a single value, then the receiver can do a check, "is this what is expected". Otherwise, it is "does it look plausible", but that requires looking at all "lanes" see so far, deciding whether this is possibly one of the random lanes, etc. We are specifying a test tool, so we should keep it simple!

I would much prefer "same initial value in all baton lanes".